tools/build-templated-pages/src/features.rs

```diff
@@ -58,53 +38,88 @@ fn parse_features(panic_on_missing: bool) -> Vec<Feature> {
             if key == "default" {
                 None
             } else {
-                let name = key
-                    .as_repr()
-                    .unwrap()
-                    .as_raw()
-                    .as_str()
-                    .unwrap()
-                    .to_string();
-                if let Some(description) = key.leaf_decor().prefix() {
-                    let description = description.as_str().unwrap().to_string();
-                    if !description.starts_with("\n# ") || !description.ends_with('\n') {
-                        panic!("Missing description for feature {name}");
+                let key_name = get_key_name(key);
+                let key_prefix = key.leaf_decor().prefix();
+                if what_to_run.contains(Command::CHECK_MISSING) {
+                    if let Some(description) = key_prefix {
+                        create_feature(&default, key_name, description)
+                    } else {
+                        panic!("Missing description for feature {key_name}");
                     }
-                    let description = description
-                        .strip_prefix("\n# ")
-                        .unwrap()
-                        .strip_suffix('\n')
-                        .unwrap()
-                        .to_string();
-                    Some(Feature {
-                        is_default: default.contains(&name),
-                        name,
-                        description,
-                    })
-                } else if panic_on_missing {
-                    panic!("Missing description for feature {name}");
                 } else {
-                    None
+                    let feature = |description| create_feature(&default, key_name, description);
+                    key_prefix.map(feature).unwrap()
```

Comment:
```suggestion
                    key_prefix.map(|description| create_feature(&default, key_name, description)).unwrap()
```

This can be inlined. There's no point it creating a variable first when it gets immediately consumed.

Reply:
The point was to reduce the number of rows by preventing row wrapping.

Reply:
I'm not sure that's a problem here. `rustfmt` handles most of these scenarios, and can make the closure multi-line if necessary:

```rust
key_prefix.map(|description| {
    create_feature(&default, key_name, description)
}).unwrap()
```

Reply:
I am used to having my refactorings judged negatively on how many rows are added, thus I compensated by creating these extra variables to prevent wrapping.
I also prefer the separated style as you can read one small statement at a time instead of one larger statement all at once.